addons/loyalty_sales_team/__init__.py

```diff
@@ -0,0 +1,4 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+pass
```

Comment:
I thought it was always loaded for any installed module, but I just tested without it and it actually works fine: the module's security settings are correctly installed.
Good to know, thanks.

Reply:
I was asking about the `pass` statement, because I saw some empty `__init__` files and none with it, but I guess it does not make any difference in the end,
There is also that cli command that can be useful when initializing new modules https://www.odoo.com/documentation/master/reference/cmdline.html#scaffolding but maybe you were already aware of it :)

Reply:
Yes, I knew about the scaffolding. But indeed, runbot does not like missing `__init__.py`:
`AssertionError: False is not true : Missing `__init__.py ` in module loyalty_sales_team`